### PR TITLE
MGMT-7775: IsDNSWildcardNotConfigured should not be part of unbound h…

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…ost validations

In case validations are being run on the unbound host, IsDNSWildcardNotConfigured should return
ValidationSuccessSuppressOutput, since it is not relevant

# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Comment by nmagnezi** [comment]
> /lgtm

Approved because it will indeed resolve the issue.
Yet, is there a room for a unit test here? (a follow up PR)

File: internal/host/validator.go [hunk 1]
```diff
@@ -1180,6 +1180,9 @@ func printIsDomainNameResolvedCorrectly(c *validationContext, status ValidationS
 }
 
 func (v *validator) isDNSWildcardNotConfigured(c *validationContext) ValidationStatus {
+	if c.infraEnv != nil {
```
Comment (filanov) [inline internal/host/validator.go:1183]: why there is not test to cover it?

**Pull request merged**